contract/test/wallet-tools.js

```diff
@@ -177,7 +187,7 @@ export const mockWalletFactory = (
      *
      * @param {Brand} brand
      */
-    async function* purseUpdates(brand) {
+    async function* purseUpdatesInternal(brand) {
```

Comment:
If this change turns out to be pointless, revert it.

Reply:
Why did you make it?